code-review-randomizer.js

```diff
@@ -0,0 +1,21 @@
+// run this script after you do a pull request and group slack the individuals listed
+
+const codeReview = yourName => {
+  const seniorEngineers = ['Inna', 'Kyle', 'Matthew', 'Patrick', 'Sayer']
+  let peopleToReviewYourCode = [];
+
+  const randomizer = () => {
+    const engineerIndex = Math.floor(Math.random() * Math.floor(seniorEngineers.length));
+    if (seniorEngineers[engineerIndex] !== yourName) {
+      peopleToReviewYourCode.push(seniorEngineers[engineerIndex]);
+    }
+    if (peopleToReviewYourCode.length < 2) {
+      randomizer();
+    }
+  }
+
+  randomizer();
+  return peopleToReviewYourCode;
+}
+
+codeReview('yourName')
```

Comment:
looks like style guide wants there to be an empty line at the bottom of the code block